pynicotine/gtkgui/widgets/clipboard.py

```diff
@@ -17,6 +17,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from gi.repository import Gdk
+from gi.repository import GObject
 from gi.repository import Gtk
```

Comment:
this import is only required if GTK_API_VERSION == 3

Reply:
It's already imported elsewhere, so it doesn't matter.